style.css

```diff
@@ -614,16 +734,68 @@ button {
     align-items: center;
   }
 
-  .menu {
-    display: block;
+  .toogle-button {
+    display: flex;
   }
 
-  .menu-desktop {
+  .overlay.active {
+    display: flex;
+  }
+
+  .overlay li {
+    flex-direction: row;
+    align-items: start;
+    list-style: none;
+  }
+
+  .skills {
+    width: 100%;
+    height: 500px;
+    flex: none;
+    order: 1;
+    margin: auto;
+  }
+
+  .overlay li a {
+    text-decoration: none;
+    color: rgb(245, 242, 242);
+    padding: 1rem;
+    display: flex;
+  }
+
+  .overlay ul li {
+    margin-bottom: 5px;
+    font-size: 32px;
+    text-transform: uppercase;
+    padding-bottom: 8px;
+  }
```

Comment:
- [ ] Kindly consider setting a `font-weight` for your `mobile-menu` text as its weight is currently not matching with that of the `Figma` design. This will make your design look close to `Figma` design template as much as possible and also create a good user experience.

Reply:
- [ ] As requested by the last reviewer, Please add the correct font-weight value for the mobile menu options to match the given Figma design.